.golangci.yml

```diff
@@ -72,6 +66,12 @@ linters:
   - godox
   - paralleltest
   - wrapcheck
+  - errorlint
+  - goerr113
+  - gomnd
+  - nestif
+  - funlen
+  - gosec
```

Comment:
:-(
Maybe add an issue to re-enable major linters?
IMO `gosec` is pretty important.
Why did you disable it? I suggest to have punctual `// nolint:` comments.